### DISCUSSION TITLE
Why did you change the design?

### DISCUSSION THREAD

**Discussion by ridesirat** [mdn/mdn-community#54] [Platform] [answered]
It was one of the best ever and everywhere.
Intemporal!
Darkmode, ok. But the design.. Noooo
Thanks for listening  :)
(no changing back, I suppose?)

**Comment by bsmth** [comment, marked as answer]
Sorry you're sad to see the old design gone! There's no plans to change back for the foreseeable future, but thanks for sharing 🙏🏻
